api/app/signals/apps/api/v1/filters/signal.py

```diff
@@ -147,6 +147,7 @@ def directing_department_filter(self, queryset, name, value):
 
         if 'null' in choices and len(choices) == 1:
             # "?directing_department=null" will select all parent Signals without a directing department
+            # we either have a relation with empty departments or no 'directing' relation at all
```

Comment:
I would rephrase this a bit, you are saying its value can be `[]` or `None` (in Python land)?

Reply:
old comments, its not applicable anymore